docs/guide/doc-ops.rst

```diff
@@ -0,0 +1,71 @@
+DocOps
+======
+
+author: Jodie Putrino & the Write the Docs community
+
+What is DocOps, anyway?
+-----------------------
+
+Along with the advent of :doc:`/guide/docs-as-code`, the technical writing community 
+has seen the growth of a new specialization called "DocOps" (or "doc-ops").
+
+But what does DocOps actually mean, and who actually "does" DocOps?
+
+This subject came up for discussion during the WTD NA 2021 Unconference session. 
+This page represents some of the ideas presented during that session.
+
+Generally speaking, "DocOps" is like `DevOps`_. Instead of applying broadly to 
+software development, though, DocOps specifically applies to the creation, 
+management, and release of documentation.  
+
+Going from the DevOps definition provided by Atlassian linked in the previous 
+paragraph, then, we could loosely define DocOps as:
+
+"a set of practices that works to automate and integrate the process 
```

Comment:
Can this be a blockquote instead of just a quoted paragraph?

Reply:
I actually couldn't figure out the directive to use for that with Sphinx. Any tips @ericholscher ?

Reply:
Just an indent I think. https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#block-quotes